backend/app/core/config.py

```
@@ -0,0 +1,32 @@
+import os
```

Ruff (0.12.2)

`os` imported but unused

Remove unused import: `os`

(F401)